docs/guides/hooks.md

```diff
@@ -0,0 +1,85 @@
+# Hooks
+Hooks exist so that users can latch on to certain Video.js lifecycle events
```

Comment:
Fair enough, but they are conceptually similar in the sense that you're providing a function to be called when a certain condition is met. But, of course, I'm not married to any particular phrasing. 😄